Makefile

```diff
@@ -0,0 +1,93 @@
+RUSTUP = rustup
+
+TOOLCHAIN = $(shell cat rust-toolchain)
+CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+
+NIGHTLY_TOOLCHAIN = "nightly-2019-07-31"
+CARGO_NIGHTLY = $(RUSTUP) run --install $(NIGHTLY_TOOLCHAIN) cargo --color always
+
+GIT_HOOKS_PATH = ".githooks"
+GIT_HOOKS = $(wildcard $(GIT_HOOKS_PATH)/*)
+
+BUILD_ARGS = ""
```

Comment:
I am not a makefile expert, can you test that indeed `-e` would override this?

Reply:
I tested it :)
`make` actually has a very useful `-n` option to dry-run a task!